src/styles/filters-history.scss

```diff
@@ -18,15 +18,30 @@
 
 .filters-history-open {
 	@include clickable;
-	background-image: url(../../resources/icons/filters-history-open.svg);
-	width: 24px;
-	height: 25px;
+	width: 40px;
+	height: 40px;
 	border: none;
+	border-radius: 12px;
+
+	display: flex;
+	justify-content: flex-start;
+	align-items: center;
+	padding-top: 10px;
+	margin-right: -5px; 
```

Comment:
I dont think you need it, cause we have gap:12px in parent div, also try to use margin, padding and gaps in future to be multiple of 4